src/server/server.rs

```diff
@@ -60,6 +60,7 @@ impl Server<AddrIncoming, ()> {
     ///
     /// This method will panic if binding to the address fails. For a method
     /// to bind to an address and return a `Result`, see `Server::try_bind`.
+    #[cfg_attr(docsrs, doc(cfg(feature = "tcp")))]
```

Comment:
Hm, do you know if we put this on the `impl Server` above, right next to the actual `cfg` attribute, will it still show nicely in the docs?

Reply:
oh yeah, the `cfg_feature` macro works fine.